### PR TITLE
docs(RBAC): Update images and update release management to include scope

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: 02.Overview/12.Role.Based.Access.Control/docs.md [hunk 1]
```diff
@@ -64,20 +62,19 @@ permissions.
...
-* **Manage**: write access to the devices, requires **Read**.
-* **Deploy**: permission to deploy software to the devices, requires **Read**.
-* **Connect**: troubleshooting access to the devices, requires **Read**.
-* **Configure**: permission to get and set devices configuration, requires **Read**.
+* **Device Group Management** restrict the access a user has to devices.
+  The scope of the permission can be limited to specific static groups of devices.
+  * **Read**: read access to the devices.
+  * **Manage**: write access to the devices, requires **Read**.
+  * **Deploy**: permission to deploy software to the devices, requires **Read**.
+  * **Connect**: troubleshooting devices (using [mender-connect](/add-ons/overview)), requires **Read**.
```
Comment (estenberg) [inline 02.Overview/12.Role.Based.Access.Control/docs.md:76]: I don't think the link will work in different branches? It was just quick and dirty for me, I think you need to prefix with ../../ or something like that, should be tested..

**Review by estenberg — CHANGES_REQUESTED** [review]

File: 02.Overview/12.Role.Based.Access.Control/docs.md (hunk 1 above)
Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: @estenberg we have a check called "test:internal-links" and it was passing; I though this one is checking those links...
anyway, I updated the link and checked locally that it's working

Comment (kjaskiewiczz) [reply 02.Overview/12.Role.Based.Access.Control/docs.md]: hmm, now it's failing

**Review by estenberg — APPROVED** [review]
:+1:

**Pull request merged**